Onboarding.md

```diff
@@ -0,0 +1,108 @@
+# Onboarding: ec-go-sdk
+
+## Purpose
+This document is a collection of all onboarding related information, tips & 
+tricks, etc. for first-time SDK contributers.
+
+## Learning Resources
+The internet contains a plethora of learning resources for the Go language. 
+Below are sources that we find useful:
+- [Go Get Started Guide](https://go.dev/learn/)
+    - Also lists additional learning resources.
+- [Pluralsight (License Required)](https://www.pluralsight.com/)
+    - We recommend the [Getting Started With Go](https://app.pluralsight.com/library/courses/getting-started-with-go/) course for an excellent guide to setting up your 
+    local development environment.
+- [Go by Example](https://gobyexample.com/)
+    - Ideal for those who learn best by doing.
+- [The Go Playground](https://go.dev/play/)
+    - Sandbox for playing with Go code.
+- [Go Docs](https://golang.org/doc/)
+    - Landing page for many Go resources e.g. learning, concepts, etc.
+- [Effective Go](https://go.dev/doc/effective_go)
+    - Outlines community-accepted code style and best practices.
+- [Our Go Guide](https://github.com/EdgeCast/ec-sdk-go/blob/main/Go.md)
+    - Best practices and code style our team adheres to.
+
+## Local Environment Setup
+If you have a Pluralsight license, we recommend you watch the Getting Started 
```

Comment:
If you have a Pluralsight license, we recommend `watching` the Getting Started

Reply:
Fixed